server/schedulers/hot_region.go

```diff
@@ -243,21 +243,21 @@ func calcScore(storeItems map[uint64][]*statistics.HotSpotPeerStat, cluster sche
 			storeStat, ok := stats[storeID]
 			if !ok {
 				storeStat = &statistics.HotRegionsStat{
-					RegionsStat: make(statistics.RegionsStat, 0, storeHotRegionsDefaultLen),
+					RegionsStat: make([]statistics.HotPeerStat, 0, storeHotRegionsDefaultLen),
 				}
 				stats[storeID] = storeStat
 			}
 
-			s := statistics.HotSpotPeerStat{
+			s := statistics.HotPeerStat{
+				StoreID:        storeID,
 				RegionID:       r.RegionID,
-				FlowBytes:      uint64(r.Stats.Median()),
 				HotDegree:      r.HotDegree,
-				LastUpdateTime: r.LastUpdateTime,
-				StoreID:        storeID,
 				AntiCount:      r.AntiCount,
+				BytesRate:      uint64(r.RollingBytesRate.Median()),
```

Comment:
I am a little confused about the name of BytesRate and RollingBytesRate

Reply:
What "confused" you?

Reply:
I think it may be a little unclear, "RollingBytesRate" seems to refer to using a sliding window to count. Is that true?

Reply:
The `RollingBytesRate` use rolling statistics to denoise. `BytesRate` is just a simple value.